### PR TITLE
ci: fixes and improvements for start_test.sh

### PR DESCRIPTION
**Pull request by altergui**
fix the check_gw_is_up loop that has been broken for a long time but no-one noticed, it waited 40 seconds unnecessarily

### CODE REVIEW

**Comment by altergui** [comment]
also, abort all tests when one of them fails. not only saves time, it also allows inspecting the relevant logs.

bash is horrible so i did my best to implement this idea

File: dockerfiles/testsuite/start_test.sh [hunk 1]
```diff
@@ -165,42 +163,36 @@ for test in ${tests_to_run[@]}; do
...
-	else
-		log "### Vochain test $test failed! ###"
-		log "### Post run logs ###"
-		$COMPOSE_CMD logs --tail 1000
-		failed="$test"
-		break
-	fi
-done
+wait_jobs() {
+	while [ -n "$(jobs)" ]; do
```
Comment (lucasmenendez) [inline dockerfiles/testsuite/start_test.sh:168]: just to learn, where the variable `jobs` is defined?

**Review by lucasmenendez — APPROVED** [review]

**Comment by p4u** [comment]
please rebase

File: dockerfiles/testsuite/start_test.sh (hunk 1 above)
Comment (altergui) [reply dockerfiles/testsuite/start_test.sh:168]: i can't recommend you to learn bash :see_no_evil:
but here we go: `jobs` is not a var but a command

**Pull request merged**